server/lib/todolist/todolist.closeTask.js

```diff
@@ -0,0 +1,19 @@
+const { ServiceNotConfiguredError } = require('../../utils/coreErrors');
+
+/**
+ * @description close task of the todolist.
+ * @param {string} taskId - The task id.
+ * @example
+ * gladys.todolist.closeTask('taskId');
+ */
+function closeTask(taskId) {
+  const todoistService = this.service.getService('todoist');
```

Comment:
I'm a bit confused...
It looks like you add the "task feature" in Gladys core, but it uses only the `todoist` specific service.

I think it  a good think to add a "todo list" abstraction in Gladys, but it is not an abstraction here.
I think you miss the "service" in your APIs...
I may be wrong of course...

Reply:
This is the same way of doing as we do in the "message" and "weather" integration, today it only uses one integration, but just because nobody developed another one ^^

I don't think it's necessarily wrong!

Reply:
@atrovato @callemand I've read a bit the code of the integration, and I still see one downside, it seems this integration was entirely designed around the todoist API, meaning that the data model is exactly todoist data model, without any changes

What about when we want to develop another integration? We translate everything to todoist format?